Classes/Query/Modifier/Faceting.php

```diff
@@ -101,7 +101,6 @@ public function modifyQuery(Query $query)
      */
     protected function buildFacetingParameters()
     {
-        $facetParameters = array();
```

Comment:
I would agree, it is not used and also scrutinizer & phpstorm mark is at unused code.